src/GenerateBatchVmSkus/appsettings.json

```diff
@@ -0,0 +1,16 @@
+{
+  "SubscriptionId": "594bef42-33f3-42df-b056-e7399d6ae7a0",
+  "OutputFilePath": "BatchSupportedVmSizeInformation.json",
+  "BatchAccounts": [
+    "msfttest52268c",
+    "msftscfc205327",
+    "bmuseast2",
+    "mattmclcoatest"
```

Comment:
did you mean to check this in?

Reply:
`SubscriptionId` and `OutputFilePath` were moved from `src/GenerateBatchVmSkus/Properties/launchSettings.json`. But I could be persuaded that this file should be stored elsewhere and used by whomever runs this tool going forward.